modules/account-lib/src/coin/sol/utils.ts

```diff
@@ -0,0 +1,86 @@
+import { Keypair, PublicKey } from '@solana/web3.js';
+import bs58 from 'bs58';
+import BigNumber from 'bignumber.js';
+
+const ENCODED_BLOCK_HASH_LENGTH = 44;
+const DECODED_BLOCK_HASH_LENGTH = 32;
+const ENCODED_ADDRESS_LENGTH = 44;
+const DECODED_ADDRESS_LENGTH = 32;
+const DECODED_SIGNATURE_LENGTH = 64;
+
+/** @inheritdoc */
+export function isValidAddress(address: string): boolean {
+  try {
+    return (
+      !!address &&
+      new RegExp('[1-9A-HJ-NP-Za-km-z]').test(address) &&
```

Comment:
this regex is used in multiple places, let's define as a constant above and reuse